README.md

```diff
@@ -34,22 +34,34 @@ The hooks (`useResponsive` and `useIsMobile`) are the preferred method of using
-| Prop               | Type    | Description                                                                   |
-| ------------------ | ------- | ----------------------------------------------------------------------------- |
-| initialMediaType   | string  | Used to mock a breakpoint in the initial state calculation. Defaults to 'xs'. |
-| defaultOrientation | string  | Used to mock orientation in initial state calculation. Defaults to null.      |
-
-## Object returned by the consumers:
-
-| Key                    | Type    | Description                                                                                                                      |
-|------------------------|---------|----------------------------------------------------------------------------------------------------------------------------------|
-| isCalculated           | Boolean | False on first render. Once true, it means all breakpoints values are based on the window.                                       |
-| mediaType              | String  | The current media type (breakpoint) name (ex: `'md'`).                                                                           |
-| lessThan               | Object  | Object containing if current window width is less than a breakpoint (usage: `lessThan.lg`).                                      |
-| greaterThan            | Object  | Object containing if current window width is greater than a breakpoint (usage: `greaterThan.lg`).                                |
-| is                     | Object  | Object containing if current window width is the at a breakpoint (usage: `is.lg`).                                               |
-| orientation            | String  | Current browser orientation (portrait, landscape or defined defaultOrientation at ResponsiveProvider, when others not available) |
-
+| Prop               | Type    | Required | Default | Description                                                                   |
+| ------------------ | ------- | ---------|---------|-------------------------------------------------------------------------------|
+| initialMediaType   | '_initial' <br>&#124;&nbsp; 'xs' <br>&#124;&nbsp; 'sm' <br>&#124;&nbsp; 'md' <br>&#124;&nbsp; 'lg'  <br>&#124;&nbsp;  'xl'| no    | 'xs'       |  Intial media type before the calculation of the real measures  |
+| defaultOrientation | 'landscape' <br>&#124;&nbsp; 'portrait'  | no    | null       | Intial oritation before the calculation of the real measures    |
+| children           | ReactNode  | yes     | -       | React component |
+| breakpoints        | { xs: string, sm: string, md: string, lg: string, xl: string }  | no    | -       | Min breakpoints     |
+| breakpointsMax     | { xs: string, sm: string, md: string, lg: string, xl: string }  | no    | -       | Max breakpoints |
+| mediaQueries       | TODO  | no    | -       | Represents the screen media queries `(If this is passed breakpointsMax is obsolete)`  |
```

Comment:
Can anyone tell me whats the value of the prop `mediaQueries` tried to understand it but thing get what I need to pass. @SoaresMG @sofiacteixeira